alf/optimizers/optimizers.py

```diff
@@ -22,6 +23,67 @@
 from . import adam_tf
 
 
+def rbf_func(x):
```

Comment:
It seems to me this function and `score_func` should be private. You can prefix the function name with '_' to indicate they are not public API. And they will not be shown in the doc site.

Reply:
Updated.